accepted/system-range/system-range.md

```diff
@@ -0,0 +1,170 @@
+# System.Range
+
+**PM** [Immo landwerth](https://github.com/terrajobst)
+
+The C# team is considering providing language syntax for specifying ranges to
+represent parts of a string, array, or span. In order to make this work, the
+language has to be bound to a .NET type, which we call `System.Range`.
+
+## Scenarios and User Experience
+
+### Constructing and enumerating ranges
+
+```csharp
+// Ranges are constructed with an index and a length
+
+var fiveToTen = new Range(index: 5, length: 5);
```

Comment:
If the length is 5, should it be fiveToNine?

Reply:
Fixed